README.md

```diff
@@ -93,19 +93,20 @@ Use and set the Default pulic token as your `MAPBOX_API_KEY` in the `local.env`f
 
 > These instructions assume Windows users are not using the WSL, which has documented problems with Docker's bind mounts. Installing and configuring Docker for Windows to work with the WSL is outside the scope of this document.
 
-## Load the dataset
+## Load a dataset
 
-Download this SQL file and place it in the projects `data` directory:
+Download one of the two SQL files and place it in the projects `data` directory:
 
-* https://sua-datafiles.s3-us-west-2.amazonaws.com/sua_20190803.sql
+* https://sua-datafiles.s3-us-west-2.amazonaws.com/sua_lane_20190915.sql - Lane County (121MB)
+* https://sua-datafiles.s3-us-west-2.amazonaws.com/sua_20190803.sql - OR, WA, ID (4.3GB, requires Docker being allocated 8GB RAM and an SSD is recommended)
 
 > Contributors: If you update this file make sure to change the filename and
 > update all references in this document.
```

Comment:
There's one more spot to update, now that there are two files. "this file" should be made plural. Feel free to reword this example replacement: "If you update one of these files or add a new one, make sure to update all references to the file (or list of files) in this document. If you're updating an existing file, make sure to change its filename to reflect the date it was generated."

Reply:
Should be resolved.